waku/common/databases/db_postgres/pgasyncpool.nim

```diff
@@ -8,15 +8,11 @@ else:
 import
   std/[sequtils,nre, strformat],
   stew/results,
-  chronicles,
   chronos
 import
   ./dbconn,
   ../common
 
-logScope:
-  topics = "postgres asyncpool"
```

Comment:
We don't log anything anymore? Or was it not used.

Reply:
Good point! It was not used. In fact, all the exposed `procs` return `Result` so it is pretty likely that we won't need many longs in this file.